pkg/operator/controllers/metrics.go

```diff
@@ -24,6 +24,14 @@ var (
 		},
 		[]string{"BuildName"},
 	)
+	GameServersStandByReconcileDuration = registry.NewGaugeVec(
```

Comment:
(related to a previous comment) Is this metric related to how much time it takes to create a GameServer? If so, we should call it in a similar fashion, like "gameserver_create_duration" or something similar. Apologies if I'm missing something!

Reply:
So, knowing that it is measuring the performance of the control plane itself, with respect to how long it takes to create the number of GameServer objects required to reach the desired state, does "GameServerBatchCreationDuration" make more sense?